<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>518</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>518</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ctdiskautodl</string>
  </property>
  <widget class="QWidget" name="userinterface">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4">
     <layout class="QGridLayout" name="block_response">
      <item row="1" column="1">
       <widget class="QPushButton" name="response_savetofile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>SAVE TO FILE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="response_shell">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QFormLayout" name="block_setting">
      <item row="0" column="0">
       <widget class="QLabel" name="ctdiskurl_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>CTDISK URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ctdiskurl_value"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_analyze">
        <property name="text">
         <string>Analyze</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="destination_label">
        <property name="text">
         <string>Destination</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="destination_value_container">
        <item>
         <widget class="QLineEdit" name="destination_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="destination_value_choosedir">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="itemnum_label">
        <property name="text">
         <string>Item #</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="itemnum_value_container">
        <item>
         <widget class="QSpinBox" name="itemnum_value_first">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="itemnum_value_separator">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="itemnum_value_last">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="timeout_label">
        <property name="text">
         <string>Timeout</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="timeout_value">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>3600</number>
        </property>
        <property name="singleStep">
         <number>30</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="button_download">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Download!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
